### ISSUE TITLE
Copy/Move studies between patients broken

### ISSUE THREAD

**Issue by vrindanayak** [dcm4che/dcm4chee-arc-light#541]
```
2017-02-02 18:34:45,023 ERROR [io.undertow.request] (default task-95) UT005023: Exception handling request to /dcm4chee-arc/aets/DCM4CHEE/rs/studies/2.25.89575827536596560410682561948727316130/move/113039^DCM: org.jboss.resteasy.spi.UnhandledException: java.lang.NumberFormatException: For input string: "42872Patient ID in incoming object does not match with that of patient associated with study."
```
Introduced by #506

Workaround: set [Accept Conflicting Patient ID](http://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/archiveDevice.html#dcmacceptconflictingpatientid) = YES

**Comment by gunterze** [comment]
@vrindanayak : `org.jboss.resteasy.spi.UnhandledException: java.lang.NumberFormatException: For input string:` is also unexpected

**Comment by vrindanayak** [comment]
Fixed #543
